gulpfile.js

```diff
@@ -7,26 +7,81 @@
  * Code distributed by Google as part of the polymer project is also
  * subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
  */
-var fs = require('fs');
-var gulp = require('gulp');
-var jshint = require('gulp-jshint');
-var path = require('path');
-var lazypipe = require('lazypipe');
-var mocha = require('gulp-mocha');
-var rollup = require('rollup');
-var runSequence = require('run-sequence');
+'use strict';
```

Comment:
Did you make any changes from what's in tools-common? Let's keep track of that and upstream. Very soon I want to make that an library that we can import rather than copy and paste.